src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -124,6 +125,18 @@ Status TSTabletManager::Init() {
   LOG(INFO) << LogPrefix(kSysCatalogTabletId) << "Bootstrapping tablet";
   TRACE("Bootstrapping tablet");
 
+  // Raft config is necessary for initializing the master tablet.
```

Comment:
this will likely be overridden by future diffs. This can be stop gap

Reply:
Just a hack to make the server work for now.